main.py

```diff
@@ -99,5 +100,49 @@ def get_destination():
     except Exception as e:
         return jsonify({'error': 'Please try again'}), 500
 
+@app.route('/get_verdict', methods=['GET'])
+def get_verdict():
+    try:
+        args = request.args
+        destination_name = args.get('destination')
+
+        destination = Destinations.query.filter_by(Name=destination_name).first()
+        if not destination:
+            return jsonify({'error': 'Destination not found'}), 404
+
+        ideal_temperature = destination.Temperature
+
+        api_key = 'KaFIkTNjT3CgIbkFGYLY3633FiW71u67'
```

Comment:
So this is sensitive data, like the db password. It would be best practice to store the api key in config and push the blank string of it to github keeping that value locally.

Obviously you can document in the readme they need to set this value and get it from the io weather site

Reply:
Sorted.